.pipelines/cni/k8s-e2e/k8s-e2e-job-template.yaml

```diff
@@ -20,11 +20,11 @@ jobs:
           addSpnToEnvironment: true
           inlineScript: |
             set -e
-            make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(make revision)
```

Comment:
we will need to keep the `-$(make revision)` in the cluster name in order to create unique clusters in each pipeline run. we won't be able to run parallel builds without it.

Reply:
It is passed in within the aks/e2e-job template.

Reply:
thanks for clarifying